packages/edit-site/src/components/sidebar/index.js

```diff
@@ -81,7 +82,10 @@ function SidebarScreens() {
 			<SidebarScreenWrapper path="/patterns">
 				<SidebarNavigationScreenPatterns />
 			</SidebarScreenWrapper>
-			<SidebarScreenWrapper path="/:postType(wp_template|wp_template_part)/all">
+			<SidebarScreenWrapper path="/:postType(wp_template)/all">
```

Comment:
The PR that switches templates parts implementation to dataviews has landed. Can this be updated to work for both?